apps/website/src/components/Outline.tsx

```diff
@@ -1,19 +1,39 @@
 'use client';
 
+import { VscChevronLeft } from '@react-icons/all-files/vsc/VscChevronLeft';
+import { Button } from 'ariakit/button';
+import { useNav } from '~/contexts/nav';
 import { useOutline } from '~/contexts/outline';
 import { Scrollbars } from './Scrollbars';
 import { TableOfContentItems } from './TableOfContentItems';
 
 export function Outline() {
-	const { members } = useOutline();
+	const { members, opened, setOpened: setOutlineOpened } = useOutline();
+	const { opened: navOpened, setOpened } = useNav();
 
 	if (!members) {
 		return null;
 	}
 
 	return (
 		<div className="lg:sticky lg:top-23 lg:h-[calc(100vh_-_105px)]">
-			<aside className="fixed bottom-4 left-4 right-4 top-22 z-20 mx-auto hidden max-w-5xl border border-light-900 rounded-md bg-white/75 shadow backdrop-blur-md lg:sticky lg:block lg:h-full lg:max-w-xs lg:min-w-xs lg:w-full dark:border-dark-100 dark:bg-dark-600/75">
+			<aside
+				className={`fixed block bottom-4 top-22 z-20 mx-auto max-w-5xl border border-light-900 rounded-md bg-white/75 shadow backdrop-blur-md transition-all duration-300 ${
+					opened ? 'left-4 right-4 ' : 'left-full -right-[calc(100vw_-_32px)]'
+				} lg:sticky lg:block lg:h-full lg:max-w-xs lg:min-w-xs lg:w-full dark:border-dark-100 dark:bg-dark-600/75`}
```

Comment:
This doesn't make any sense.

`block` and `lg:block` are both applied at the same time here basically, you also removed the `hidden` class if its not opened.